lte/gateway/c/oai/tasks/amf/amf_security_mode_control.cpp

```diff
@@ -432,6 +432,18 @@ int amf_proc_security_mode_control(
       snni[30] = 0x72;  // r
       snni[31] = 0x67;  // g
 
+      // int r = sprintf(
+      //   snni, "5G:mnc%01d%01d%01d.mcc%01d%01d%01d.3gppnetwork.org",
+      //  (char*) plmn.mnc_digit1, (char*) plmn.mnc_digit2,
+      // (char*) plmn.mnc_digit3, (char*) plmn.mcc_digit1,
+      //  (char*) plmn.mcc_digit2, (char*) plmn.mcc_digit3);
+      //  if (r != 32) {
+      //    OAILOG_ERROR(LOG_NAS_AMF, "failed to construct SNN!\n");
+      //  OAILOG_FUNC_RETURN(LOG_NAS_AMF, RETURNerror);
+      // } else {

```

Comment:
remove the unused code.

Reply:
why do you have (char *) casting in the commented code..

Reply:
this code not required, removed